akamai/gtm-liveness-test/README.md

```diff
@@ -2,92 +2,92 @@
 
 To secure your IBM Cloud Kubernetes Service or Red Hat OpenShift on IBM Cloud cluster, you might use [Calico pre-DNAT network policies](https://cloud.ibm.com/docs/containers?topic=containers-network_policies), [VPC security groups](https://cloud.ibm.com/docs/containers?topic=containers-vpc-network-policy#security_groups), [VPC access control lists (ACLs)](https://cloud.ibm.com/docs/containers?topic=containers-vpc-network-policy#acls), or another custom firewall solution to block incoming traffic to Ingress or router services.
 
-To ensure that the Kubernetes or OpenShift control plane can check the health of your ALBs or routers, you must allow inbound access from the following Akamai addresses and ports.
+To ensure that the Akamai Global Traffic Management liveness test can check the health of your ALBs or routers, you must allow inbound access from the following Akamai addresses. You must allow inbound access on the port based on the protocol which is configured for the liveness test. The default protocol is HTTP which means port 80. If you configure HTTPS the corresponding port is 443. 
```

Comment:
`You must allow inbound access on the port based on the protocol which is configured for the liveness test.` how do users configure a protocol for the liveness test? is there something users are supposed to do? Or, does it just mean, they need to know whether their ALBs/routers can listen for HTTP or HTTPS requests

Reply:
Users can configure the liveness test for their ALBs as described in this doc: https://cloud.ibm.com/docs/containers?topic=containers-loadbalancer_hostname#loadbalancer_hostname_monitor

The logic is: if there are more than a single IPs for the same cluster subdomain, IBM Cloud Kubernetes Service automatically creates liveness test for that subdomain with protocol=HTTP and port=80. 
On top of that users can also manage liveness tests as described in the document above.

Reply:
Ah okay, I see. Then what about something like this?

```suggestion
To ensure that the Akamai Global Traffic Management liveness test can check the health of your ALBs or routers, you must allow inbound access from the following Akamai IP addresses. To use the default HTTP protocol, allow inbound traffic through port 80. If you configure your ALBs or routers to listen for the HTTPS protocol, such as with the [`ibmcloud ks nlb-dns monitor configure` command](https://cloud.ibm.com/docs/containers?topic=containers-cli-plugin-kubernetes-service-cli#cs_nlb-dns-monitor-configure), allow inbound traffic through port 443. 
```

Reply:
Yep, that is great. Fixed. Thank you!